lib/dock.rb

```diff
@@ -0,0 +1,46 @@
+class Dock
+
+  attr_reader :name,
+              :max_rental_time,
+              :boat_inventory,
+              :total_renters,
+              :total_rental_by_person
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @boat_inventory = {}
+    @total_renters = []
+    @log_hour = 0
+    @total_rental_by_person = {}
+  end
+
+  def add_boats_to_inventory(type)
```

Comment:
Based on the spec, and the rest of the functionality you have here, you probably don't need to have boats added to an inventory collection - it makes sense, but is unnecessary for this challenge